FieldOpt/Optimization/objective/NPV.h

```diff
@@ -64,6 +65,8 @@ class NPV : public Objective {
   QList<Component *> *components_; //!< List of gamma, k pairs.
   Simulation::Results::Results *results_;  //!< Object providing access to simulator results.
   Settings::Optimizer *settings_;
+  Model::Model::Economy *well_economy_;
+  Model::Model *model_;
```

Comment:
Is it necessary to keep the model? Isn't WellEconomy enough?

Reply:
Yupp